app/controllers/practices_controller.rb

```diff
@@ -364,6 +277,9 @@ def published
   end
 
   def publication_validation
+    pr_params = { practice: @practice, practice_params: practice_params }
+    SavePracticeService.new(pr_params).save_practice
```

Comment:
if this returns `false` what happens?
do we still try to publish?
should we still try to publish? :thinking:

Reply:
Oh... 😓 Nothing happens when `false` is returned from saving the practice...
My inclination is if there's an issue with saving the practice, we shouldn't try to publish....
If they saved successfully and they meet the requirements for publishing, then they should be allowed to publish. Do you have any opinions on this?

Reply:
i agree